pkg/implementer_test.go

```diff
@@ -0,0 +1,107 @@
+package impl
+
+import (
+	"bytes"
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func getTestFile() (*token.FileSet, *ast.File) {
+	fs := token.NewFileSet()
+
+	file, _ := parser.ParseFile(fs, "./test/testmain.go", nil, 0)
```

Comment:
There are quite a lot of hard-coded slashes in file paths in this PR. Please fix them.

Also, when opening files, the leading `./` is unnecessary. That's only to execute the file in POSIX shell.

Reply:
Yeah, the `./` is just me being lazy and not cleaning up. My vim setup auto-completes filenames after I type `./`, and I just haven't bothered to clean them up.

By cleaning up hard-coded file slashes, do you mean you prefer `path.Join` instead?